css/pages/index.css

```diff
@@ -3,50 +3,122 @@
 }
 
 .container {
-  width: 750px;
+  width: 100%;
   text-align: center;
   font-size: 1.5rem;
 }
 
+.second-screen {
+  width: 100%;
+}
+
 .cards-title {
-  margin-bottom: 32px;
-  color: #e20665;
+  margin-top: 1041px;
```

Comment:
1041px.... Such a huge value... Where is this being used and why?

Reply:
Oops I was just doing some random stuff and forget to remove this while pushing the changes.